bgpd/bgp_updgrp.c

```diff
@@ -1482,7 +1482,24 @@ static int updgrp_policy_update_walkcb(struct update_group *updgrp, void *arg)
 					"u%" PRIu64 ":s%" PRIu64" announcing default upon default routemap %s change",
 					updgrp->id, subgrp->id,
 					ctx->policy_name);
-			subgroup_default_originate(subgrp, 0);
+			if (route_map_lookup_by_name(ctx->policy_name)) {
```

Comment:
I think this is already done in `updgrp_route_map_update()`, right? (`def_changed`)

Reply:
Yeah, but in case of policy change, update flow was not being triggered,
since the `SUBGRP_STATUS_DEFAULT_ORIGINATE` was not unset when we call `subgroup_default_originate`.
Hence, in order to invoke the update flow, unsetting the flag only if the associated route-map is found.